cro-clib/Cargo.toml

```diff
@@ -6,7 +6,9 @@ edition = "2018"
 
 [lib]
 name = "cro_clib"
-crate-type = ["staticlib"]
+#in ios, android, use staticlib
+#crate-type = ["staticlib"]
```

Comment:
remove this line, might confuse the sed script in `docker/build.sh`

Reply:
ok, i'll check build.sh

Reply:
also, will inspect travis